git_cli_vs_gui/git_cli_vs_gui.md

```diff
@@ -0,0 +1,337 @@
+* SourceTree
+* GitHub Desktop
+* and many more!
+
+Some of these are free, some are free and open source, and some have a cost (either for everyone, or with a "freemium" model that has a free tier).  Which is the best GUI for Git?  We'd argue that the best Git GUI is the one that your team uses and you'll have friendly support around.  
+
+## Command Line Interface and Git
+
+Many people find that using the command line interface for Git gives them more fine-grained control over their projects.  Also, if you have experience working in the command line, you may be faster and more comfortable using typed commands rather than learning the particular design of a Git GUI.  The Git commands used in a CLI are part of the core Git software, which means you don't have to pay for them, as you might with some GUI software.  Finally, if you're working on a computer system that doesn't use a graphical interface (for example, a Linux server that doesn't use a GUI or desktop), you'll still be able to use Git in the command line.
+
+To use Git in a CLI, you will use a number of commands that start with the word `git`.  For example, you might type `git status` to understand what branch you have checked out and if there are any untracked files to be aware of, or `git commit -m "My useful commit message"` to make a commit with a message you specify that gives more information about that commit.
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+We just threw some Git jargon at you: **branch**, **commit**, **checkout**, **tracked**.  Don't worry if you don't know what these mean.  The important thing we want you to be aware of is what these commands look like when you're using Git in a CLI.
+
+</div>
+
+## Which is Better? 
+
+Which is better when you use Git?  Graphical user interfaces, or a command line interface? It depends!  In the next section, we'll do some direct compare and contrast so you can get an idea of what the two approaches might look like in actual day to day work.  
+
+Generally speaking, for everyday work in Git, you can choose whichever approach appeals to you more, so we'll present both the command line interface as well as a graphical user interface to allow you to see both. However, it's important to note that whichever interface you choose for daily work, you **might end up needing to use a command line interface for complex, out of the ordinary work** in Git.  Let's talk a bit about why this is the case.
+
+Git has lots of features, and designers can't include every possible command in an easy-to-use point and click GUI.  That means some things you may need to do in Git can **only** be done by using the command line, because they're so niche and specific to certain circumstances that graphical software simply hasn't included that task.  Still, there's a good chance that you might end up using a Git GUI for **years** before you run across a situation that requires the command line. 
```

Comment:
@leemc-data-ed  this is a bit of a combination of the two repetitive paragraphs.  What do you think?

Reply:
This is great!